### PR TITLE
Ifaa 53 2

### PR DESCRIPTION
**Pull request by ihor-dynka**

### CODE REVIEW

File: src/test/java/com/softserveinc/edu/ita/pageobjects/OrderingPage.java [hunk 1]
```diff
@@ -82,6 +82,41 @@ public OrderingPage(WebDriver driver) {
     }
 
     /**
+     *  sets orderNames or orderStatuses or orderAssignees in list depends on parameters
+     *
+     * @param condition
+     * @return
+     */
+    public List<WebElement> getOrderFromView(OrderSearchCondition condition){
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/OrderingPage.java]: it returns web elements on columns, not sure if it a page object action

File: src/test/java/com/softserveinc/edu/ita/pageobjects/PageObjectBase.java [hunk 2]
```diff
@@ -54,6 +56,11 @@ public void sendKeys(ILocator locator, String text) {
         Reporter.log(String.format("<br>INFO&nbsp;&nbsp; - Typed '%s' in <b>%s</b>", text, locator.getName()));
     }
 
+    public void clear(ILocator locator){
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/PageObjectBase.java]: this is not a generic method, you are clearing the search field on the Ordering page, move it out of base

File: src/test/java/com/softserveinc/edu/ita/tests/ordering_page/FilterTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.softserveinc.edu.ita.enums.ordering_page.OrderFilter.ROLE;
+import static com.softserveinc.edu.ita.enums.ordering_page.OrderFilter.STATUS;
+import static com.softserveinc.edu.ita.enums.ordering_page.StatusFilterValue.NONE;
+
+public class FilterTest extends TestRunner {
+    List<WebElement> ordersList;
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/tests/ordering_page/FilterTest.java]: where are the access modifiers?

File: src/test/java/com/softserveinc/edu/ita/tests/ordering_page/SearchTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+import java.util.List;
+
+import static com.softserveinc.edu.ita.locators.OrderingPageLocators.ORDER_ASSIGNEE_COLUMN;
+import static com.softserveinc.edu.ita.locators.OrderingPageLocators.ORDER_NAME_COLUMN;
+import static com.softserveinc.edu.ita.locators.OrderingPageLocators.ORDER_STATUS_COLUMN;
+
+
+public class SearchTest extends TestRunner {
+    List<WebElement> ordersList;
```
Comment (demchyna) [inline src/test/java/com/softserveinc/edu/ita/tests/ordering_page/SearchTest.java]: palka - access modifiers

**Pull request merged**